src/panel/applets/budgie-menu/com.solus-project.budgie-menu.gschema.xml

```diff
@@ -14,6 +14,12 @@
       <description>The menu label to display on the menu button</description>
     </key>
 
+    <key type="b" name="use-default-menu-icon">
+      <default>true</default>
```

Comment:
As discussed on Matrix, let's default this to `false`, and add a check to force it on if any of the following themes are in use: default, Adwaita, hicolor

That way, all the other themes can show their icons by default, while still having a sensible icon for the other ones and still having the option to use ours.